p2p/common/messenger_testworld.go

```diff
@@ -0,0 +1,149 @@
+// +build testworld
+
+package p2pcommon
+
+import (
+	"context"
```

Comment:
like mentioned on slack we can run `goimports -w .` to fix this import order everywhere.